thirdeye-spi/src/main/java/ai/startree/thirdeye/spi/datalayer/dto/AlertMetadataDTO.java

```diff
@@ -30,6 +30,11 @@ public class AlertMetadataDTO {
   private String granularity;
   private String timezone;
   private EventContextDto eventContext;
+  /**
+   * ISO-8601 period. Eg P7D. At detection run, TE re-runs the detection starting from min(lastTimestamp,
+   * endTime-replayPeriod). Used to manage data mutability.
+   */
+  private String mutabilityPeriod;
```

Comment:
✅ Mutability period w/in the Alert makes sense to me.

Reply:
had recommendation from @suvodeep-pyne to put it in the DatasetConfigDto.
I have no strong opinion. 
For consistency I'll put it in DatasetConfigDto close to completenessDelay

Reply:
just to clarify instead of being in metadata$mutabilityPeriod it will be in metadata$dataset$mutabilityPeriod